### DISCUSSION TITLE
Retrieve alignment_heads of a fine-tuned model

### DISCUSSION THREAD

**Discussion by Ar770** [openai/whisper#1733] [Q&A]
Hi, 
I'm looking for a method to retrieve the alignment_heads of a fine-tuned Whisper model. 
I need it as I understand that the word-level timestamps depend on them. 
Thanks

**Comment by Ar770** [comment]
@jongwook can you please help with this?

**Reply by jongwook** [reply]
I think you can keep using the same alignment heads as the base model. If you're curious, save the `QKs` tensor in timing.py:

https://github.com/openai/whisper/blob/f6f01c561c45ad6ab421405e18ae22fd0c698e92/whisper/timing.py#L205

and manually inspect the heads (that's how I made the masks at least...).
